src/Gui/TreeView.cpp

```diff
@@ -47,10 +47,7 @@ TreeView::TreeView(QWidget* parent)
     this->setDropIndicatorShown(false);
     this->setRootIsDecorated(false);
     this->setSelectionMode(QAbstractItemView::ExtendedSelection);
-#if QT_VERSION >= 0x040200
-    // causes unexpected drop events (possibly only with Qt4.1.x)
     this->setMouseTracking(true); // needed for itemEntered() to work
-#endif
 }

```

Comment:
Should we investigate if this still is needed? (the comment suggests it might be a qt 4.1.x only issue)

Reply:
My read of that comment is that `this->setMouseTracking(true);` is needed, but had problems with Qt 4.1. It's currently compiled.

Reply:
My bad, I misread the `#if`